packages/cicero-core/test/data/formatted-dates-NOFORMAT/sample.txt

```diff
@@ -0,0 +1 @@
+dateTimeProperty: 31/12/2019
```

Comment:
Confused, shouldn't this have to be 12/31/2019 ?

Reply:
Bogged my test when I tried to make it fail. Thanks!

Reply:
Fixed now.